frontend/src/pages/modelServing/__tests__/useInferenceServices.spec.ts

```diff
@@ -1,17 +1,17 @@
 import { act } from '@testing-library/react';
 import { k8sListResource } from '@openshift/dynamic-plugin-sdk-utils';
-import { InferenceServiceKind } from '~/k8sTypes';
 import { standardUseFetchState, testHook } from '~/__tests__/unit/testUtils/hooks';
 import useModelServingEnabled from '~/pages/modelServing/useModelServingEnabled';
 import useInferenceServices from '~/pages/modelServing/useInferenceServices';
 import { useAccessReview } from '~/api';
+import { mockK8sResourceList } from '~/__mocks__/mockK8sResourceList';
+import { mockInferenceServiceK8sResource } from '~/__mocks__/mockInferenceServiceK8sResource';
+import { InferenceServiceKind } from '~/k8sTypes';
 
-const mockInferenceServices = {
-  items: [
-    { metadata: { name: 'item 1' } },
-    { metadata: { name: 'item 2' } },
-  ] as InferenceServiceKind[],
-};
+const mockInferenceServices = mockK8sResourceList([
```

Comment:
not the scope of this PR , but can make use of  existing mock function 
```suggestion
const mockInferenceServices = mockK8sResourceList([mockInferenceServiceK8sResource({})]);
```

Reply:
This change already uses `mockInferenceServiceK8sResource` but keeps two entries because that's what the previous test was doing. The scope of this PR isn't to re-evaluate and improve all tests.